learner_pathway_progress/migrations/0005_auto_20221007_0758.py

```diff
@@ -0,0 +1,27 @@
+# Generated by Django 3.2.15 on 2022-10-07 12:58
+
+from django.db import migrations
+
+
+def apply(apps, schema_editor):
+    LearnerPathwayMembership = apps.get_model("learner_pathway_progress", "LearnerEnterprisePathwayMembership")
```

Comment:
nit: Use single quotes instead of double quotes for strings inside the file.

Reply:
Changing double quotes to single quotes.